evalml/pipelines/pipeline_base.py

```diff
@@ -759,7 +759,9 @@ def _supports_fast_permutation_importance(self):
             for c in self.component_graph
         )
         has_dfs = any(isinstance(c, DFSTransformer) for c in self.component_graph)
-        if has_dfs:
+        # If the pipeline is a stacked ensemble, the DFS Transformer name will be different
+        # and we don't need to check for the pre_existing features anyway
+        if has_dfs and not has_more_than_one_estimator:
```

Comment:
The other option for fixing was to change `self.component_graph[DFSTransformer.name]` to use whatever name is actually being used for the dfs transformer(s), but I don't know of any other way the name could be anything other than `"DFS Transformer"` other than the stacked ensemble case, so this seemed simpler.